Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM python:3.9.5-slim-buster as base
+
```

Comment:
You should set a WORKDIR, so you are explicitly copying files into a particular location in the container. 

E.g. `WORKDIR /app`